test/ex_machina/ecto_test.exs

```diff
@@ -43,14 +43,18 @@ defmodule ExMachina.EctoTest do
     end
   end
 
-  test "params_with_assocs/2 inserts belongs_to associations" do
+  test "params_with_assocs/2 inserts belongs_tos that are set by the factory" do
+    assert Enum.member?(ExMachina.Article.__schema__(:associations), :editor)
```

Comment:
I like this, but I think extracting to a private function would be good. Something like `assert has_association_in_schema?(ExMachina.Article, :editor)`. What do you think?